solr/core/src/java/org/apache/solr/client/solrj/embedded/JettySolrRunner.java

```diff
@@ -436,7 +433,7 @@ protected HandlerWrapper injectJettyHandlers(HandlerWrapper chain) {
    * @return the {@link CoreContainer} for this node
    */
   public CoreContainer getCoreContainer() {
-    if (getSolrDispatchFilter() == null || getSolrDispatchFilter().getCores() == null) {
+    if (getSolrDispatchFilter() == null) {
```

Comment:
I'd still worry about test code that calls this and races before its set.

Reply:
The line of code change here was a simplification that should work the same way as before.  If SDF.getCores() is null, then the line below will return null just the same ;-)